tikv-configuration-file.md

```diff
@@ -1157,7 +1144,7 @@ raftdb 相关配置项。
 ### `max-background-jobs`
 
 + RocksDB 后台线程个数。
-+ 默认值：2
++ 默认值：4
 + 最小值：1
```

Comment:
This is also 2?

Reply:
According to Tikv code, it's set to 4.
<img width="641" alt="Screen Shot 2021-03-05 at 14 29 04" src="https://user-images.githubusercontent.com/26134513/110076249-251fd200-7dbf-11eb-9693-c758225e1715.png">
<img width="637" alt="Screen Shot 2021-03-05 at 14 29 53" src="https://user-images.githubusercontent.com/26134513/110076334-4385cd80-7dbf-11eb-8b55-2547ab10bc5e.png">

Reply:
I mean the minimal size. Due to `cmp::max(2, ...)`, I think it's actually 2.